site2/docs/helm-install.md

```diff
@@ -4,34 +4,8 @@ title: Install Apache Pulsar using Helm
 sidebar_label: "Install "
 ---
 
-Install Apache Pulsar on Kubernetes with the official Pulsar Helm chart.
+````mdx-code-block
+import {Redirect} from '@docusaurus/router';
```

Comment:
FYI redirections should follow this pattern https://github.com/apache/pulsar/pull/18415#issue-1443829113.

Reply:
Thanks for the reminder. Done :)